routes/licenses.test.js

```diff
@@ -28,4 +28,22 @@ describe('license routes', () => {
       expect(response.payload).toMatchSnapshot();
     });
   });
+
+  describe('GET /license', () => {
+    function options() {
+      return { url: `/license/File:Pommes-1.jpg`, method: 'GET' };
+    }
+
+    async function subject() {
+      return context.inject(options());
+    }
+
+    it('returns a list of licenses', async () => {
```

Comment:
Minor, since we'll update this anyways, but it's only a **single** license object here 😉